pyeasytries/Trie.py

```diff
@@ -69,7 +99,7 @@ def add(self, word_to_add):
         >>> trie.add("firetruck")
         TRUE 
         """
-        
```

Comment:
Could you undo this change? This is part of a line for `add()` so I worried this line might create merge conflict later when we are trying to merge Jordan's PR

Reply:
Sure on it!

Reply:
Undone! Thanks Kaicheng!